app/models/post.rb

```diff
@@ -1,2 +1,24 @@
+require 'csv'
+require 'prawn'
 class Post < ApplicationRecord
+  
+  belongs_to :user
+  
+  def self.done
+    where(done_homework: true)
+  end
+
+  def self.pending
+    where(done_homework: false)
+  end
```

Comment:
@8geonirt @diegosaul402 should these methods be declared as scopes? (self.done and self.pending)